### PR TITLE
CAAPIFeedbackLink

### PR DESCRIPTION
**Pull request by videor**
Adding CA API Feedback Tip and link.

### CODE REVIEW

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -1390,36 +1390,40 @@
         "id": "7d3116a3-1690-4a15-ba47-153cb7891d00",
         "category": "Security",
         "method": "GET",
-        "humanName": "get all conditional access policies",
+        "humanName": "List Conditional Access policies",
```
Comment (bettirosengugi) [inline sample-queries/sample-queries.json:1393]: human name should start with a small letter

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -1390,36 +1390,40 @@
...
         "docLink": "https://docs.microsoft.com/en-us/graph/api/conditionalaccessroot-list-policies?view=graph-rest-1.0&tabs=http",
+        "tip": "We’d like to hear from you. Please leave your feedback on Conditional Access API here: https://aka.ms/caapifeedback",
         "skipTest": false
     },
     {
         "id": "4fc821e4-f291-4944-999a-6d5b84ec8735",
         "category": "Security",
         "method": "GET",
-        "humanName": "get all named locations",
+        "humanName": "List Named Locations",
```
Comment (bettirosengugi) [inline sample-queries/sample-queries.json:1403]: human name should start with a small letter.

File: sample-queries/sample-queries.json [hunk 3]
```diff
@@ -1390,36 +1390,40 @@
...
         "docLink": "https://docs.microsoft.com/en-us/graph/api/conditionalaccessroot-list-namedlocations?view=graph-rest-1.0&tabs=http",
+        "tip": "We’d like to hear from you. Please leave your feedback on Conditional Access API here: https://aka.ms/caapifeedback",
         "skipTest": false
     },
     {
         "id": "473bd812-0df6-4fcb-bd23-d4b81eaa5f3e",
         "category": "Security",
         "method": "GET",
-        "humanName": "get all conditional access policies (beta)",
+        "humanName": "List Conditional Access policies (beta)",
```
Comment (bettirosengugi) [inline sample-queries/sample-queries.json:1413]: human name should start with a small letter.

File: sample-queries/sample-queries.json [hunk 4]
```diff
@@ -1390,36 +1390,40 @@
...
         "docLink": "https://docs.microsoft.com/en-us/graph/api/conditionalaccessroot-list-policies?view=graph-rest-beta&tabs=http",
+        "tip": "We’d like to hear from you. Please leave your feedback on Conditional Access API here: https://aka.ms/caapifeedback",
         "skipTest": false
     },
     {
         "id": "d565b5f3-a268-4acf-a068-565294b5cdae",
         "category": "Security",
         "method": "GET",
-        "humanName": "get all named locations (beta)",
+        "humanName": "List Named Locations (beta)",
```
Comment (bettirosengugi) [inline sample-queries/sample-queries.json:1423]: human name should start with a small letter.

**Review by bettirosengugi — CHANGES_REQUESTED** [review]

File: sample-queries/sample-queries.json (hunk 4 above)
Comment (videor) [reply sample-queries/sample-queries.json]: @bettirosengugi  I have made the changes.

**Comment by sanana30** [comment]
Hi @bettirosengugi - Vikas has made the changes required. Can you please review?

**Review by bettirosengugi — APPROVED** [review]

**Pull request merged**